examples/energy-management-app/energy-management-common/energy-evse/include/EnergyEvseDelegateImpl.h

```diff
@@ -208,6 +208,12 @@ class EnergyEvseDelegate : public EnergyEvse::Delegate
      */
     Status ScheduleCheckOnEnabledTimeout();
 
+    /**
+     * @brief   Helper function to handle timer expiration when in enabled state
+     * @param matterEpoch Current time in Matter epoch seconds
+     */
+    void HandleEnabledStateExpiration(uint32_t matterEpoch);
```

Comment:
```suggestion
    void HandleEnabledStateExpiration(uint32_t matterEpochSeconds);
```

Or use a Seconds32 if you want to make it clear that it's seconds, either way.